app/routers/audio.py

```diff
@@ -0,0 +1,376 @@
+
+    return json.dumps(
+        {"music_on": music_on,
+            "playlist": playlist,
+            "music_vol": music_vol,
+            "sfxs_on": sfxs_on,
+            "sfx_choice": CHOSEN_SFX,
+            "sfxs_vol": CHOSEN_SFX_VOL})
+
+
+# Functions for audio setup according to users' choices:
+
+
+def init_audio_tracks(
+        session: Session,
+        song_names: List[str],
+        sfx_names: List[str]):
+    """This function fills the AudioTracks table
+
+    Args:
+        session (Session): the database
+        song_names (List[str]): list of song names
+        sfx_names (List[str]): list of sfxs' names.
+    """
+    for song in song_names:
+        add_audio_track(session, song, True)
+    for sfx in sfx_names:
+        add_audio_track(session, sfx, False)
+
+
+def add_audio_track(session: Session, title: str, is_music: bool):
+    """Adds a new audio track to AudioTracks table.
+
+    Args:
+        session (Session): the databse.
+        title (str): title of the track.
+        is_music (bool): if set to true - music, sfx otherwise.
+    """
+    res = session.query(AudioTracks).filter_by(title=title).first()
+    if not res:
+        track = AudioTracks(title=title, is_music=is_music)
+        session.add(track)
+        session.commit()
+
+
+def get_tracks(
+    session: Session,
+    user_id: int
+        ) -> Tuple[Optional[List[str]], Optional[str]]:
+    """Retrieves audio selections from the database,
+    for both music and sound effects.
+
+    Args:
+        session (Session): the database.
+        user_id (int): current users' id.
+
+    Returns:
+        Tuple[Optional[List[str]], Optional[str]]:
+        returns the playlist of music tracks, as well as sound effect choice.
+    """
+    playlist = []
+    chosen_track_ids = session.query(
+        UserAudioTracks.track_id).filter_by(user_id=user_id)
+    tracks = session.query(
+        AudioTracks).filter(
+            AudioTracks.id.in_(chosen_track_ids)).filter_by(is_music=1)
+    sfx = session.query(
+        AudioTracks).filter(
+            AudioTracks.id.in_(chosen_track_ids)).filter_by(is_music=0).first()
+    for track in tracks:
+        playlist.append(track.title + ".mp3")
+    sfx_choice = sfx.title + ".wav" if sfx else None
+
+    return playlist, sfx_choice
+
+
+def get_audio_settings(
+    session: Session, user_id: int = 1
+) -> Tuple[Optional[List[str]], Optional[int], Optional[str], Optional[int]]:
+    """Retrieves audio settings from the database.
+
+    Args:
+        session (Session): [description]
+        user_id (int, optional): [description]. Defaults to 1.
+
+    Returns:
+        Tuple[str, Optional[List[str]], Optional[int],
+        str, Optional[str], Optional[int]]: the audio settings.
+    """
+    music_on, music_vol, sfxs_on, sfxs_vol = None, None, None, None
+    playlist, sfx_choice = get_tracks(session, user_id)
+    audio_settings = session.query(
+        AudioSettings).filter_by(user_id=user_id).first()
```

Comment:
We should have some united UserSettings table. Can you please talk with @PureDreamer regarding the best way to save the users' settings?